viewer/vueapp/src/components/spigraph/Spigraph.vue

```diff
@@ -65,7 +65,7 @@
             <div class="input-group input-group-sm">
               <span class="input-group-prepend cursor-help"
                 v-b-tooltip.hover
-                title="Sort results by">
+                title="Chose SPIGraph Type">
```

Comment:
I'm terrible at english, but should this be choose or chosen?

Reply:
Chosen

Reply:
Oh, yes. "Chosen SPIGraph Type" :)